src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const PRICE_IN_DAY = 40;
+
+  let price = days * PRICE_IN_DAY;
+
+  if (days >= LONG_TERM) {
+    price = price - LONG_TERM_DISCOUNT;
+  }
```

Comment:
Instead of reassigning `price`, you can return the result immediately within the `if` block to simplify the function flow. This will also allow you to use `const` for `price` as suggested earlier.